charts/opentelemetry-operator/templates/service.yaml

```diff
@@ -12,6 +12,10 @@ spec:
       port: 8443
       protocol: TCP
       targetPort: https
+    - name: metrics
```

Comment:
Since service monitor is optional, this should also be optional.

Reply:
@TylerHelmuth As long as the container is exposing that port and endpoint by default I think it's good practice to propagate it at the `Service` level. This is not needed just for the ServiceMonitor but it's a way to access metrics from the pod/s behind the service by any other application/client.

Reply:
I agree.  Can you move both this change and the new containerPort into a new PR?  Those 2 changes seem to fix an issue that the metrics port is not exposed and it should be.